pom.xml

```diff
@@ -44,8 +44,8 @@
     <groovyVersion>3.0.5</groovyVersion>
     <atlasVersion>0.17.1</atlasVersion>
 
-    <pmeVersion>4.1</pmeVersion>
-    <gmeVersion>2.3-SNAPSHOT</gmeVersion>
+    <pmeVersion>4.2</pmeVersion>
+    <gmeVersion>2.3</gmeVersion>
```

Comment:
Can we revert this portion as it clashes with https://github.com/project-ncl/manipulator-groovy-examples/pull/20

Reply:
It won't compile without the change I think. If you commit #20, then I can rebase.

Reply:
Ugh ... that snapshot repo has disappeared. I'd rather have yours committed then I rebase on top. If you want to split out your test changes thats up to you.

Reply:
Yes, fine, I will split the test change into a separate PR, leaving this just the yml file.

Reply:
Great, perfect, thanks!